mlflow/utils/databricks_utils.py

```diff
@@ -439,34 +468,64 @@ def get_databricks_host_creds(server_uri=None):
         MlflowHostCreds which includes the hostname and authentication information necessary to
         talk to the Databricks server.
     """
+    # Since we do not record OAuth expiration time in OAuth file, perform periodic refresh of OAuth cache here.
+    # As currently configured (02/24) OAuth token in model serving environment guaranteed to have at least 30 min 
+    # remaining on TTL at any point in time but refresh at higher rate here to be safe and in case those values change in the future. 
+    OAUTH_CACHE_REFRESH_DURATION = 10 * 60
+    OAUTH_CACHE_ENV_VAR = "DATABRICKS_DEPENDENCY_OAUTH_CACHE"
+    OAUTH_CACHE_EXPIRATION_ENV_VAR = "DATABRICKS_DEPENDENCY_OAUTH_CACHE_EXIRY_TS"
+
     profile, path = get_db_info_from_uri(server_uri)
     config = ProfileConfigProvider(profile).get_config() if profile else get_config()
```

Comment:
will we actually be able to get config in model serving environment?

Reply:
if we don't have config is there any otherway to get host

Reply:
We can populate it in Model Serving as an env var

@sueann - any objections to adding envrionment variable for workspace url in serving containers?